CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.3.0
+
+- `non_constant_identifier_names` updated to check local variables, for-loop
+  initializers and catch clauses
+- error range of `lines_longer_than_80_chars` updated to start at 80 (to make)
```

Comment:
Not sure why there are parens here. Consider removing them.

Reply:
odd! me neither!  🤔 

thanks!